framework/i18n/Formatter.php

```diff
@@ -787,6 +787,50 @@ public function asRelativeTime($value, $referenceTime = null)
         }
     }
 
+    /**
+     * Formats the value as a time interval.
+     * @param mixed $value the value to be formatted, in ISO8601 format.
```

Comment:
As for me, ISO8601 is not convenient to format intervals.
I'd prefer simple format - in seconds.